Readme.md

```diff
@@ -181,6 +181,30 @@ next(Nope.)
 completed
 ```
 
+#### mapAt
+
+Replace every element with its value at the given key path.
```

Comment:
Its not urgent, so I don't mind merging without change, but I think the term replacing is really confusing - map is a transform method, I would expect something like 

"Transform every element to its value at a given keypath" 

If we change it , then we'll later change this on mapTo as well since I don't think the replace word is appropriate